website/styles/login.css

```diff
@@ -0,0 +1,74 @@
+@import "unified.css";
+
+/* Bordered form */
+form {
+    border: 3px solid #f1f1f1;
+}
+
+/* Full-width inputs */
+input[type=email], input[type=password] {
+    width: 100%;
```

Comment:
100% width looks fine on mobile, but a little weird on desktop. Consider using media queries to have the properties change on different sized devices. 
![image](https://user-images.githubusercontent.com/8786151/46537997-486a8e00-c8a2-11e8-89e1-85f6cded0303.png)

Reply:
Will do.